.travis.yml

```diff
@@ -2,20 +2,22 @@
 language: ruby
 bundler_args: --without development
 before_install: rm Gemfile.lock || true
-sudo: false
+sudo: true
 rvm:
-#  - 1.8.7
   - 1.9.3
   - 2.0.0
   - 2.1.0
-script: bundle exec rake test
+script: if [ $BEAKER == true ]; then make beaker; else bundle exec rake test; fi
 env:
-#  - PUPPET_VERSION="~> 2.7.0"
   - PUPPET_VERSION="~> 3.1.0"
   - PUPPET_VERSION="~> 3.3.0"
   - PUPPET_VERSION="~> 3.7.4" FUTURE_PARSER=yes STRICT_VARIABLES=yes
   - PUPPET_VERSION="~> 4.0.0"
   - PUPPET_VERSION="~> 4.1.0"
+  - BEAKER=true
+  global:
```

Comment:
may be the `PUPPET_VERSION=*` need to move into `global` as well? Currently it seems to conflict between the 2 data types; array and hash.